Dictionary renderer with feature layer (#736)
Esri/arcgis-runtime-samples-ios: arcgis-ios-sdk-samples/Layers/Dictionary renderer with feature layer/README.md

# Dictionary renderer with feature layer

Convert features into graphics to show them with mil2525d symbols.

![Dictionary renderer with feature layer](dictionary-renderer-with-feature-layer.png)

## Use case

A dictionary renderer uses a style file along with a rule engine to display advanced symbology. This is useful for displaying features using precise military symbology.

## How to use the sample

> When launched, the sample will display military symbology. Pan and zoom to explore.

Reviewer: The design has "Simply run the sample." for this section. But since all samples are contained within a single app, neither "launch" nor "run" are appropriate verbs. Maybe something like this?

```suggestion
This sample displays military symbology. Pan and zoom to explore the map.
```

@mi
